### ISSUE TITLE
Problems with non-latin album names

### ISSUE THREAD

**Issue by mosfet07** [lcharlick/python-metallum#9]
album_for_id() returns correct data:
```
metallum.album_for_id(104072).title
-> 'Äpäre'
```
but band_for_id().albums[].title doesn't:
```
metallum.band_for_id(2052).albums[6].title
-> 'Ã\x84pÃ¤re'
```

Same when writing to a file.

It's interesting that after I concatenate the album's band name with the album title, album.title starts to work fine:
```
a = metallum.band_for_id(2052).albums[6]
a.title
-> 'Ã\x84pÃ¤re'
a.bands[0].name + a.title
-> 'AjattaraÄpäre'
a.title
-> 'Äpäre'
```

Windows 7 64-bit, Python 3.8.1

**Comment by lcharlick** [comment]
Thanks for reporting, should be fixed with dc9a02dac889fe19910d02ae8be47a674d2d834b

**Comment by mosfet07** [comment]
Thanks mate. I confirm it works fine now.
